app/lib/features/chat_ng/widgets/chat_bubble.dart

```diff
@@ -106,12 +108,15 @@ class ChatBubble extends StatelessWidget {
               child: child,
             ),
           ),
-          Visibility(
-            visible: wasEdited,
-            child: Text(
-              L10n.of(context).edited,
-              style: chatTheme.emptyChatPlaceholderTextStyle
-                  .copyWith(fontSize: 12),
+          Align(
+            alignment: Alignment(0.9, 0.0),
+            child: Visibility(
+              visible: wasEdited,
+              child: Text(
+                L10n.of(context).edited,
+                style: chatTheme.emptyChatPlaceholderTextStyle
+                    .copyWith(fontSize: 12),
+              ),
```

Comment:
Unnecessary wrapping in `Visibility` ...

```suggestion
          if (wasEdited)
          Align(
            alignment: Alignment(0.9, 0.0),
            child: Text(
              L10n.of(context).edited,
              style: chatTheme.emptyChatPlaceholderTextStyle
                  .copyWith(fontSize: 12),
```